src/test/java/com/puppycrawl/tools/checkstyle/api/TokenTypesTest.java

```diff
@@ -59,33 +64,23 @@ public void testAllDescriptionsEndsWithPeriod() {
         assertEquals(Collections.emptySet(), badDescriptions, "Malformed TokenType descriptions");
     }
 
-    @Test
-    public void testGetShortDescription() {
-        assertEquals(
-                "The <code>==</code> (equal) operator.",
-                TokenUtil.getShortDescription("EQUAL"), "short description for EQUAL");
-
-        assertEquals(
-                "The <code>&&</code> (conditional AND) operator.",
-                TokenUtil.getShortDescription("LAND"), "short description for LAND");
-
-        assertEquals(
-                "A left curly brace (<code>{</code>).",
-                TokenUtil.getShortDescription("LCURLY"), "short description for LCURLY");
-
-        assertEquals(
-                "The <code>>>=</code> (signed right shift assignment) operator.",
-                TokenUtil.getShortDescription("SR_ASSIGN"), "short description for SR_ASSIGN");
-
-        assertEquals(
-                "The <code><<</code> (shift left) operator.",
-                TokenUtil.getShortDescription("SL"), "short description for SL");
-
-        assertEquals(
-                "The <code>>>></code> (unsigned shift right) operator.",
-                TokenUtil.getShortDescription("BSR"), "short description for BSR");
+    @ParameterizedTest(name = "{index} => expected: ''{0}'', token: ''{1}''")
+    @MethodSource("testGetShortDescriptionProvider")
```

Comment:
it is not good approach.
it is very compact and very technological but VERY complicated and cryptic.
We need easy to read tests and make them as much as possible to be focused on human ability to read them easily - https://github.com/checkstyle/checkstyle/wiki/Checkstyle-GSoC-2021-Project-Ideas#project-name-adaptation-of-behavior-driven-development-ideas-for-testing-of-source-code-validation-algorithms

---
 

I do not understand why we need so much instances for `getShortDescription`, better approach is simply remove all except for one.